js/mage/adminhtml/uploader/instance.js

```diff
@@ -104,11 +104,16 @@
             this.elementsIds = config.elementIds;
             this.elements = this.getElements(this.elementsIds);
 
+            // Fix error where setting post_max_size or upload_max_filesize to 0
+            // causes the flow.js to make infinite chunks and crash the browser
+            if(config.uploaderConfig.chunkSize === 0) {
+                config.uploaderConfig.chunkSize = Number.POSITIVE_INFINITY;
+            }
+
             this.uploaderConfig = config.uploaderConfig;
             this.browseConfig = config.browseConfig;
             this.miscConfig =  config.miscConfig;
```

Comment:
I would suggest to add this line roughly here to enforce orderly updates:
`this.uploaderConfig.simultaneousUploads = 1;`

What happened in my case: the contend department uploaded a bunch of previously ordered images which ended up being shuffeled by the uploader because of the multiple requests.
I did not test the performance thoroughly but I noticed no slowdown of the upload.

If you feel not comfortable to add this line I will create my own PR, but I thougt to myself that a PR with one line is overkill 😅

Reply:
I would highly recommend adding this line. Especially nowadays, due to better internet connections, it shouldn't be a problem to upload the files one after the other and get a reliable/expected order of the files.